### PR TITLE
Phase 11

### PR DESCRIPTION
**Pull request by matindaghyani**

### CODE REVIEW

File: Phase-5/SearchWebAPI/Controllers/SearchController.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using SearchEngine.Classes.IO.Database.Models;
+using SearchEngine.Interfaces.Core;
```
Comment (nazari02105) [inline Phase-5/SearchWebAPI/Controllers/SearchController.cs]: I Actually really love this comment:
useless imports :)))))) (just kidding)
it is not really important but it is good to remove useless imports

File: Phase-5/SearchWebAPI/Controllers/SearchController.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        private readonly ISearchEngineCore _searchEngineCore;
+
+        public SearchController(ISearchEngineCore searchEngineCore)
+        {
+            _searchEngineCore = searchEngineCore;
+        }
+
+        [HttpGet()]
+        public List<int> Query(string toSearch)
```
Comment (nazari02105) [inline Phase-5/SearchWebAPI/Controllers/SearchController.cs]: IDE give this fact about this HttpGet():
Parentheses are redundant if attribute has no arguments

File: Phase-5/SearchWebAPI/Program.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
```
Comment (nazari02105) [inline Phase-5/SearchWebAPI/Program.cs]: ahhh Here you go again :)

**Review by nazari02105 — CHANGES_REQUESTED** [review]
some tiny comments

**Review by nazari02105 — APPROVED** [review]

**Review by nazari02105 — APPROVED** [review]
Well Done

**Review by mohammad258852 — APPROVED** [review]

**Review by alrz1999 — APPROVED** [review]

**Pull request merged**